src/System.Linq/ref/System.Linq.cs

```diff
@@ -182,6 +182,7 @@ public static partial class Enumerable
         public static System.Collections.Generic.HashSet<TSource> ToHashSet<TSource>(this System.Collections.Generic.IEnumerable<TSource> source) { throw null; }
         public static System.Collections.Generic.HashSet<TSource> ToHashSet<TSource>(this System.Collections.Generic.IEnumerable<TSource> source, System.Collections.Generic.IEqualityComparer<TSource> comparer) { throw null; }
         public static System.Collections.Generic.List<TSource> ToList<TSource>(this System.Collections.Generic.IEnumerable<TSource> source) { throw null; }
+        public static System.Collections.Generic.List<TSource> ToList<TSource>(this System.Collections.Generic.IEnumerable<TSource> source, int capacity) { throw null; }
```

Comment:
This changes the public API so will need an [API review](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/api-review-process.md) first.